.travis.yml

```diff
@@ -1,11 +1,9 @@
 language: node_js
 node_js:
+  - "0.12"
```

Comment:
Can we add `io.js` as well? It should be:

```yaml
- "iojs"
```

http://docs.travis-ci.com/user/languages/javascript-with-nodejs/